wzdx/raw_to_standard/icone.py

```diff
@@ -19,18 +21,28 @@ def main():
     generated_messages = generate_standard_messages_from_string(
         input_file_contents)
 
-    generated_files_list = []
+    wzdx_msg = {}
     for message in generated_messages:
-        output_path = f"{output_dir}/standard_icone_{message['event']['source']['id']}_{round(message['rtdh_timestamp'])}.json"
-        open(output_path, 'w+').write(json.dumps(message, indent=2))
-        generated_files_list.append(output_path)
-
-    if generated_files_list:
-        print(
-            f"Successfully generated standard message files: {generated_files_list}")
-    else:
-        logging.warning(
-            "Standard message generation failed. See messages printed above")
+        wzdx = icone_translator.wzdx_creator(message)
+        if not wzdx_msg:
+            wzdx_msg = wzdx
+        else:
+            wzdx_msg['features'].extend(wzdx['features'])
+
+    # generated_files_list = []
```

Comment:
Was this meant to be pulled back in? If not, let's clean out unused code.

Reply:
Updated